iib/workers/tasks/build.py

```diff
@@ -669,7 +671,11 @@ def _prepare_request_for_build(
     :param list add_arches: the list of arches to build in addition to the arches ``from_index`` is
         currently built for; if ``from_index`` is ``None``, then this is used as the list of arches
         to build the index image for
-    :param list bundles: the list of bundles to create the bundle mapping on the request
+    :param list bundles: the list of bundles to create the bundle mapping on the request.
+    :param str source_from_index: the pull specification of the container image containing the index
+        that will be used as a base of the merged index image.
+    :param str target_from_index: the pull specification of the container image containing the index
```

Comment:
This doesn't seem right, the source_from_index data will be added here and a few more. Could you please reword this?

Reply:
Hopefully it's more accurate now.